README.md

```diff
@@ -85,20 +89,33 @@ utils/dbdump/create_db.sql) but then you need to change the database name in
 src/main/resources/hibernate.default.properties) accordingly.
 
 ## Create the sink FHIR store and BigQuery dataset
-First you need to create a GCP project. Then create a Google
-Cloud Healthcare dataset and a BigQuery dataset with the same name in that
-project (for an overview of projects, datasets and data stores check [this](
-https://cloud.google.com/healthcare/docs/concepts/projects-datasets-data-stores)
-document). Once that is done, use the script
-[`utils/create_fhir_store.sh`](utils/create_fhir_store.sh) to create
+To set up GCP project that you can use as a sink FHIR store:
+
+1. Create a new project in [GCP](https://console.cloud.google.com). For an overview of projects, datasets and data stores check [this document](https://cloud.google.com/healthcare/docs/concepts/projects-datasets-data-stores).
```

Comment:
I don't think `+` or `-` turn in to numbered lists, at least in git-flavored md

Reply:
You are right, I mostly wanted to avoid keeping the numbers in the text, apparently we can have them all as `1`according to [this](https://www.markdownguide.org/basic-syntax/#ordered-lists); marking as resolved.